### PR TITLE
Fix supply items recipes

### LINKED ISSUE #2214: 1.10.2 - NO recipes for creating Supply Camp or Supply Ship
### Minecolonies version

minecolonies-universal-1.10.2-0.9.7375.jar

### Expected behavior

- Create simple camp or just start new world with new colony

### Actual behaviour

- There is NO recipes for creating Supply Camp or Supply Ship so without them I can not have a Townhall and can not start playing

### Steps to reproduce the problem

1. create new world
2. place 5 chests like that https://wiki.minecolonies.com/images/f/f9/Supply_Camp.png
3. ...
4. Nothing happens. It did not create Supply Camp

### PR DESCRIPTION
**Pull request by MineShadow7**
Closes #2214 

# Changes proposed in this pull request:
-Brings back supply chest and camp recipes

Review please

### CODE REVIEW

**Comment by marchermans** [comment]
@MineShadow7 Why is a test failing that succeeded in the last alpha builds for any branch?

**Comment by MineShadow7** [comment]
That's strange... I succesfully launched my build.

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java [hunk 1]
```diff
@@ -162,7 +162,17 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
 
         //Field
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutField, 1),
-                                                    " Y ", "X#X", " X ", 'X', WOODEN_STICK, '#', Items.LEATHER, 'Y', Blocks.HAY_BLOCK));
+                " Y ", "X#X", " X ", 'X', WOODEN_STICK, '#', Items.LEATHER, 'Y', Blocks.HAY_BLOCK));
+
+        //Supply Items
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModItems.supplyCamp, 1), "   ", "C C", "CCC", 'C', Blocks.CHEST));;
```
Comment (BuildTeamLinter) [inline src/main/java/com/minecolonies/coremod/util/RecipeHandler.java:168]: MINOR Remove this empty statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)

**Comment by BuildTeamLinter** [comment]
SonarQube analysis reported 42 issues
* ![MAJOR][MAJOR] 41 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [blocks#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [gui#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/client/gui#): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [render#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/client/render#): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [colony#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony#): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [buildings#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony/buildings#): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [permissions#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony/permissions#): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [requestable#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony/requestable#): Add a 'package-info.java' file to document the 'requestable' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [init#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony/requestsystem/init#): Add a 'package-info.java' file to document the 'init' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [locations#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony/requestsystem/locations#): Add a 'package-info.java' file to document the 'locations' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [management#](https://github.com/Minecolonies/minecolonies/blob/3bea510e09dd6fb197fbb88af4c8651325b83707/src/main/java/com/minecolonies/coremod/colony/requestsystem/management#): Add a 'package-info.java' file to document the 'management' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**